internal/cmd/base/servers.go

```diff
@@ -342,6 +345,29 @@ func (b *Server) RemovePidFile(pidPath string) error {
 	return os.Remove(pidPath)
 }
 
+func (b *Server) StoreWorkerAuthReq(authReq, workerAuthReqPath string) error {
+	// Quit fast if no workerAuthReqFile
+	if workerAuthReqPath == "" {
+		return nil
+	}
+	workerAuthReqFilePath := fmt.Sprintf("%s/%s", workerAuthReqPath, WorkerAuthReqFile)
```

Comment:
Use `[filepath.Join](https://pkg.go.dev/path/filepath#Join)` as this is safe across OSes.